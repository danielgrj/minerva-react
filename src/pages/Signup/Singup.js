import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  border: 1px solid black;
`;

const Signup = () => (
  <Container>
    <h1>Signup</h1>
  </Container>
);

export default Signup;
