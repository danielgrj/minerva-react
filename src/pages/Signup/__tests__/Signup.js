import React from 'react';
import { shallow } from 'enzyme';
import Signup from '..';

describe('<Signup />', () => {
  describe('render()', () => {
    it('should match snapshot', () => {
      const wrapper = shallow(<Signup />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
