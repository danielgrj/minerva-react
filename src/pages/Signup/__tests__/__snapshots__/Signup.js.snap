// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Signup /> render() should match snapshot 1`] = `ShallowWrapper {}`;
